includes/Stories_Renderer/Renderer.php

```diff
@@ -182,6 +187,34 @@ public function init() {
 		$this->story_posts = array_map( [ $this, 'prepare_story_modal' ], $this->stories->get_stories() );
 	}
 
+	/**
+	 * Return the fields state.
+	 *
+	 * @return FieldState
+	 */
+	public function field() {
+		$view = isset( $this->attributes['view_type'] ) ? $this->attributes['view_type'] : 'grid';
+
+		switch ( $view ) {
+			case 'grid':
+				$field_state = new GridView();
+				break;
+			case 'list':
+				$field_state = new ListView();
+				break;
+			case 'circles':
+				$field_state = new CircleView();
+				break;
+			case 'carousel':
+				$field_state = new CarouselView();
+				break;
+			default:
+				$field_state = apply_filters( 'web_stories_default_fieldstate', new GridView() );
```

Comment:
We should have a filter comment, similar to 
https://github.com/google/web-stories-wp/blob/782c4029d28b938280c80203ca8d3b222a993c0c/includes/Traits/Publisher.php#L92-L103